crates/api/src/site/mod_log.rs

```diff
@@ -98,13 +241,75 @@ impl Perform for GetModlog {
     ) = if data.community_id.is_none() {
       blocking(context.pool(), move |conn| {
         Ok((
-          ModRemoveCommunityView::list(conn, mod_person_id, page, limit)?,
-          ModBanView::list(conn, mod_person_id, page, limit)?,
-          ModAddView::list(conn, mod_person_id, page, limit)?,
-          AdminPurgePersonView::list(conn, mod_person_id, page, limit)?,
-          AdminPurgeCommunityView::list(conn, mod_person_id, page, limit)?,
-          AdminPurgePostView::list(conn, mod_person_id, page, limit)?,
-          AdminPurgeCommentView::list(conn, mod_person_id, page, limit)?,
+          if matches!(
+            type_,
+            ModlogActionType::All | ModlogActionType::ModRemoveCommunity
+          ) && other_person_id.is_none()
```

Comment:
wasn't in the initial plans. Do you want that feature before we merge?

Reply:
wasn't in the initial design. Do you want this feature before we merge this?

Reply:
I spose its not necessary, but add a TODO comment because this seems missing. If the recipient of an action is a post, and you allow filtering by the recipient creator, I don't see why community actions and community creators should be left out.